src/main/java/net/mcreator/ui/dialogs/MCItemSelectorDialog.java

```diff
@@ -151,6 +152,32 @@ public MCItemSelectorDialog(MCreator mcreator, MCItem.ListProvider blocksConsume
 			});
 		}
 
+		if (supportPotions) {
```

Comment:
I think this should be implemented with MCItem.ListProvider and show previews of potions in existing list, so we don't need a separate button. We can do this here as we have previews of custom and vanilla potions